assets/jsx/bagger.jsx

```diff
@@ -19,15 +19,11 @@ class Bagger extends React.Component {
     componentDidMount() {
         const { dispatch } = this.props
         const b = document.getElementById('bagger');
-        if (b.dataset.accessKeyId !== undefined) { // TODO: any? all? add fine grain actions?
-            const dataset = b.dataset
-            dispatch(BagActions.updateConfig(
-                dataset.accessKeyId,
-                dataset.secretAccessKey,
-                dataset.bucket,
-                dataset.region,
-                dataset.keyPrefix
-                ))
+        const dataset = b.dataset
+        const args = [dataset.accessKeyId, dataset.secretAccessKey, dataset.bucket, dataset.region,
+                        dataset.keyPrefix]
```

Comment:
It looks like that's indented by two extra spaces? (this reminds me – is esformatter stable on JSX? Might be worth a global format-only commit if so)